access/templates/access/create_form_default.html

```diff
@@ -3,41 +3,21 @@
 
 {% block exercise %}
 
-{% if result.accepted %}
-<div class="alert {% if result.error_fields %}alert-danger{% else %}alert-success{% endif %}">
-<p>
-    {% if exercise.accepted_message %}
-    {{ exercise.accepted_message|safe }}
-    {% else %}
-    {% blocktrans with points=result.points max_points=exercise.max_points %}
-    You got {{ points }}/{{ max_points }} points from this questionnaire.
-    {% endblocktrans %}
-    {% endif %}
-</p>
-<p>
-    {% if result.form.group_errors %}
-    {% blocktrans %}
-    Note that the whole section of questions is marked as being incorrect if
-    one of the questions in that section is incorrect.
-    {% endblocktrans %}
```

Comment:
Was there a reason for removing this warning about question groups? Does the UI otherwise imply clearly that grouped questions require correct answers on all questions? (This setting is not often used these days since the RST questionnaire format doesn't even have any syntax for defining these question groups?)

Reply:
Good point. Maybe it indeed wasn't clear enough. But on the other hand I don't think it was clear enough before either. It was like this before:

![image](https://user-images.githubusercontent.com/43036333/90253906-95c6c780-de4a-11ea-9705-62e499c05702.png)
 
But the red danger alert was removed in the previous pull request. I now implemented it somewhat back, but instead of using the red alert, I used the black line of the `alert alert-default`, and indicated that whichever questions are within those lines and form the question group that has errors, are marked incorrect. It is not ideal, but better:

![image](https://user-images.githubusercontent.com/43036333/90254386-5351ba80-de4b-11ea-9345-a3f23c76e708.png)

And like you said, since there is no directive for the group, I think (and hope) that it was not used much before.

Reply:
This was the earlier implementation alongside with the message you pointed out:
https://github.com/apluslms/mooc-grader/blob/fcdfd7fee9e3993d2f26d625afe95538cd4b1359/access/templates/access/graded_form.html#L5

Reply:
Great, now it's better!